src/imp/gui/Notate.java

```diff
@@ -17114,7 +17114,7 @@ private void adviceTreeKeyPressed(java.awt.event.KeyEvent evt) {//GEN-FIRST:even
 
                 TreeNode parent = node.getParent();
 
-                Enumeration<TreeNode> e = parent.children();
+                Enumeration<? extends TreeNode> e = parent.children();

```

Comment:
This change come from https://github.com/Impro-Visor/Impro-Visor/issues/63#issuecomment-1470963389
Which is the only compile error for modern jdk, I tested with jdk17.